operator/pkg/helmreconciler/patching.go

```diff
@@ -19,107 +19,14 @@ import (
 	"fmt"
 
 	jsonpatch "github.com/evanphx/json-patch"
-	"github.com/pkg/errors"
-	"k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/apimachinery/pkg/util/jsonmergepatch"
-	"k8s.io/apimachinery/pkg/util/mergepatch"
 	"k8s.io/apimachinery/pkg/util/strategicpatch"
-	"k8s.io/client-go/kubernetes/scheme"
-	kubectl "k8s.io/kubectl/pkg/util"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// CreatePatch creates a patch based on the current and updated versions of an object
-func (h *HelmReconciler) CreatePatch(current, updated runtime.Object) (Patch, error) {
```

Comment:
I am not sure whether this part is still needed, maybe create different PR to clean up helmreconciler?

Reply:
this is dead code, so it's not needed

Reply:
if it is, we can always recover from git history